src/main/java/org/reaktivity/nukleus/sse/internal/stream/SseServerFactory.java

```diff
@@ -755,21 +755,25 @@ private void handleEnd(
                     .payload(sseEvent.buffer(), sseEvent.offset(), sseEvent.sizeof())
                     .build();
 
-                if (networkReplyBudget >= sseEvent.sizeof() + networkReplyPadding)
+                if (networkSlot == NO_SLOT)
                 {
-                    networkReply.accept(data.typeId(), data.buffer(), data.offset(), data.sizeof());
-                    doHttpEnd(networkReply, networkRouteId, networkReplyId, traceId, authorization);
-                    cleanupDebitorIfNecessary();
+                    networkSlot = bufferPool.acquire(networkReplyId);
                 }
-                else
+
+                if (networkSlot != NO_SLOT)
```

Comment:
if `networkSlot == NO_SLOT` then we need to `ABORT` instead of deferred `END`, right?